Orange/canvas/report/report.py

```diff
@@ -162,8 +162,7 @@ def report_plot(self, name=None, plot=None):
             self.report_html += get_html_img(plot.scene())
 
     # noinspection PyBroadException
-    def report_table(self, name, table=None, header_rows=0, header_columns=0,
-                     num_format=None):
+    def report_table(self, name, table=None, header_rows=0, header_columns=0, tot_rows = 0, num_format=None):
```

Comment:
With only a few small issues left, soon! :smile:

You can [re-read the comments](https://github.com/biolab/orange3/pull/1097/files) and read between the lines. In this chunk, I'd prefer if `tot_rows` was named more explicitly. Is the parameter even necessary? Can't it be devised given `table` input?